pom.xml

```diff
@@ -206,6 +163,17 @@
                     </javacArgs>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.8.1</version>
+                <configuration>
+                    <source>${java.version}</source>
+                    <target>${java.version}</target>
+                    <skipMain>true</skipMain> <!-- skip compile -->
```

Comment:
why?

Reply:
It's originally came from zzcclp's commit and used just to skip the  compilation of java sources, because the whole project code in scala is compiled by `scala-maven-plugin`